typescript/sdk/src/hook/EvmHookModule.ts

```diff
@@ -916,6 +928,102 @@ export class EvmHookModule extends HyperlaneModule<
     return hook;
   }
 
+  // NOTE: this deploys the ism too on the destination chain if it doesn't exist
+  protected async deployCCIPHook({
+    config,
+  }: {
```

Comment:
the CCIP hook/ism deploy logic has been moved to the HyperlaneCCIPDeployer in the next PR https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/5412